packages/esm-service-queues-app/src/active-visits/active-visits-table.component.tsx

```diff
@@ -128,11 +128,11 @@ function OldQueueTable({ queueEntries }: { queueEntries: QueueEntry[] }) {
   const currentQueueLocation = useSelectedQueueLocationUuid();
   const { queues } = useQueues(currentQueueLocation);
   const { visitQueueNumberAttributeUuid } = useConfig<ConfigObject>();
-  const visitQueueEntries = queueEntries.map((entry) =>
-    mapVisitQueueEntryProperties(entry, visitQueueNumberAttributeUuid),
-  );
-
   const currentServiceUuid = useSelectedServiceUuid();
+  const visitQueueEntries = queueEntries
+    .map((entry) => mapVisitQueueEntryProperties(entry, visitQueueNumberAttributeUuid))
+    .filter((entry) => (currentServiceUuid ? entry.queue.service.uuid === currentServiceUuid : true));
+
```

Comment:
I wouldn't think this filter is needed at all.  We should just be able to use `queueEntries` as-is, since it is already filtering by both the selected service and location.  So let's get rid of this `visitQueueEntries` and just use the `queueEntries` returned from the hook.